docs/usage.md

```diff
@@ -1216,4 +2200,2474 @@ If you're ready, you can then remove the files with
+2. The path must have at least one `*` wildcard character. This is even if you
+   are only running one paired end sample.
+3. When using the pipeline with paired end data, the path must use `{1,2}` or
+   `{R1,R2}` notation to specify read pairs.
+4. If you are running Single end data make sure to specify `--single_end`
+
+**Important**: The pipeline can't take a list of multiple input files - it takes
+a 'glob' expression. If your input files are scattered in different paths then
+we recommend that you generate a directory with symlinked files. If running in
+paired end mode please make sure that your files are sensibly named so that they
+can be properly paired. See the previous point.
+
+If the pipeline can't find your files then you will get the following error
+
+```bash
+ERROR ~ Cannot find any reads matching: *{1,2}.fastq.gz
+```
+
+IF your sample name is "messy" then you have to be very particular with your
+glob specification. A file name like `L1-1-D-2h_S1_L002_R1_001.fastq.gz` can be
+difficult enough for a human to read. Specifying `*{1,2}*.gz` won't work give
+you what you want whilst `*{R1,R2}*.gz` will.
+
+### I am only getting output for a single sample although I specified multiple with wildcards
+
+You must specify paths to files in quotes, otherwise your shell will evaluate
+any wildcards (\*) rather than Nextflow.
+
+For example
+
+```bash
+nextflow run nf-core/eager --input /path/to/sample_*/*.fq.gz
+```
+
+Would be evaluated by your shell as  
+
+```bash
+nextflow run nf-core/eager --input /path/to/sample_1/sample_1.fq.gz /path/to/sample_1/sample_1.fq.gz /path/to/sample_1/sample_1.fq.gz
+```
+
+And Nextflow will only take the first path after `--input`, ignoring the others.
+
+On the other hand encapsulating the path in quotes will allow Nextflow to
+evaluate the paths.
+
+```bash
+nextflow run nf-core/eager --input "/path/to/sample_*/*.fq.gz"
+```
+
+### The pipeline crashes almost immediately with an early pipeline step
+
+Sometimes a newly downloaded and set up nf-core/eager pipeline will encounter an
+issue where a run almost immediately crashes (e.g. at `fastqc`,
+`output_documentation` etc.) saying the tool could not be found or similar.
+
+#### I am running Docker
+
+You may have an outdated container. This happens more often when running on the
+`dev` branch of nf-core/eager, because docker will _not_ update the container on
+each new commit, and thus may not get new tools called within the pipeline code.
+
+To fix, just re-pull the nf-core/eager docker container manually with:
+
+```bash
+docker pull nfcore/eager:dev
+```
+
+#### I am running Singularity
+
+If you're running singularity, it could be that nextflow cannot access your
+singularity image properly - often due to missing bind paths.
+
+See
+[here](https://nf-co.re/usage/troubleshooting#cannot-find-input-files-when-using-singularity)
+for more information.
+
+### The pipeline has crashed with an error but Nextflow is still running
```

Comment:
Seems like you have the same problem...

Reply:
What do you mean here?

Reply:
Ah, I think you can ignore that. @apeltzer did a review on our proteomicslfq and I think we discussed that problem as well. I made a "Known issue" out of it as well.

Reply:
❤️  that people are finding these and making overall progress on reporting this back to the main nf-core docs :-)